src/Functions/normalizedQueryHash.cpp

```diff
@@ -37,94 +37,10 @@ struct Impl
         ColumnString::Offset prev_src_offset = 0;
         for (size_t i = 0; i < size; ++i)
         {
-            SipHash hash;
-
             ColumnString::Offset curr_src_offset = offsets[i];
-            Lexer lexer(reinterpret_cast<const char *>(&data[prev_src_offset]), reinterpret_cast<const char *>(&data[curr_src_offset - 1]));
+            res_data[i] = normalizedQueryHash(
```

Comment:
BTW, if we extracted one function to separate file, it will be inconvenient if another similar function (normalizeQuery) is not extracted.

Reply:
Agree.